### PR TITLE
Class card images

### PR DESCRIPTION
**Pull request by andburn**
As requested by @jleclanche 
- class card frames for spells and minions (weapons stay the same).
- optional extra: inhand golden frames :smile: 

**EDIT** Removed class and golden images from preload asset queue.

### CODE REVIEW

File: ts/TexturePreloader.ts [hunk 1]
```diff
@@ -7,12 +7,19 @@ class CardArtPreloader extends Stream.Writable {
...
 	protected images = [];
 	private working = 0;
-	protected assetQueue = ['cardback', 'hero_frame', 'hero_power', 'inhand_minion', 'inhand_spell', 'inhand_weapon',
+	protected assetQueue = ['cardback', 'hero_frame', 'hero_power',
+							'inhand_minion_druid', 'inhand_minion_hunter', 'inhand_minion_mage',
+							'inhand_minion_paladin', 'inhand_minion_priest', 'inhand_minion_rogue',
+							'inhand_minion_shaman', 'inhand_minion_warlock', 'inhand_minion_warrior',
+							'inhand_minion_neutral', 'inhand_spell_druid', 'inhand_spell_hunter',
+							'inhand_spell_mage', 'inhand_spell_paladin', 'inhand_spell_priest', 'inhand_spell_rogue',
+							'inhand_spell_shaman', 'inhand_spell_warlock', 'inhand_spell_warrior', 'inhand_spell_neutral',
```
Comment (Epix37) [inline ts/TexturePreloader.ts]: I'm not sure all of these should be in the asset queue.

File: ts/components/game/Card.tsx [hunk 2]
```diff
@@ -58,6 +59,7 @@ class Card extends React.Component<CardProps, {}> {
 		var defaultHealth = null;
 		var defaultDurability = null;
 		var cardType = entity.getCardType();
+		var cardClass = entity.getClass();
```
Comment (andburn) [inline ts/components/game/Card.tsx]: I may be misunderstanding how this should work, but it seems to be always 0. Used the card data below instead.

File: ts/TexturePreloader.ts (hunk 1 above)
Comment (jleclanche) [inline ts/TexturePreloader.ts]: They shouldn't be.

Comment (andburn) [inline ts/TexturePreloader.ts]: You're probably right, no sense preloading all of them. Maybe just strip it back to the neutrals.

Comment (Epix37) [inline ts/TexturePreloader.ts]: Optimally we would want to load them dynamically based on the heroes.

**Comment by beheh** [comment]
Awesome, thanks! I'll review this in the next few days, probably tomorrow.

File: ts/components/game/Card.tsx (hunk 2 above)
Comment (beheh) [inline ts/components/game/Card.tsx]: Yeah, I have no idea why it's not tracked. It's fine for now.

**Comment by beheh** [comment]
Great work. Landed with minor adjustments as of 178881323e94e3cba0b01fdc2cc1fac169997328.

**Pull request closed without merging**